File: data/src/main/java/com/mrp/sml/data/local/SmlDatabase.java
package com.mrp.sml.data.local;

import androidx.room.Database;
import androidx.room.RoomDatabase;

@Database(entities = {TransferEntity.class}, version = 1, exportSchema = false)
public abstract class SmlDatabase extends RoomDatabase {
    public abstract TransferDao transferDao();
}

File: data/src/main/kotlin/com/mrp/sml/data/local/SmlDatabase.kt
package com.mrp.sml.data.local

import androidx.room.Database
import androidx.room.RoomDatabase

@Database(
    entities = [TransferEntity::class],
    version = 1,
    exportSchema = false,
)
abstract class SmlDatabase : RoomDatabase() {
    abstract fun transferDao(): TransferDao
}
